src/promptflow/tests/sdk_cli_test/e2etests/test_flow_test.py

```diff
@@ -275,7 +275,13 @@ def test_eager_flow_test(self):
 
     def test_eager_flow_test_with_yaml(self):
         clear_module_cache("entry")
-        flow_path = Path(f"{EAGER_FLOWS_DIR}/simple_with_yaml/").absolute()
+        flow_path = Path(f"{EAGER_FLOWS_DIR}/simple_with_yml/").absolute()
```

Comment:
Why is it still `simple_with_yml` while the test title says `with_yaml`?

Reply:
Good catch, I accidentally modified this.